public/img/donationBtnImg/blue-big.svg

```diff
@@ -0,0 +1,11 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 212 64">
+  <defs/>
+  <rect width="212" height="64" fill="url(#paint0_radial)" rx="32"/>
+  <path fill="#fff" d="M65 26.6h4.2a6 6 0 014.3 1.5c1 1 1.6 2.4 1.6 4.2 0 1.8-.5 3.2-1.6 4.2a6 6 0 01-4.3 1.5H65zm4 9.7c2.7 0 4-1.3 4-4s-1.3-4-4-4h-2v8zm14.7 1.8a4.9 4.9 0 01-4.8-2.7 7 7 0 01-.6-3.1c0-1.2.2-2.2.7-3.1.4-.9 1-1.6 1.9-2a5 5 0 012.8-.7c1.2 0 2.1.2 3 .7a4 4 0 011.8 2c.4.9.7 1.9.7 3a7 7 0 01-.7 3.2c-.4.9-1 1.6-1.9 2-.8.5-1.7.7-2.9.7zm0-1.6a3 3 0 002.5-1c.6-.8.9-1.9.9-3.2 0-1.4-.3-2.4-.9-3.1a3 3 0 00-2.5-1 3 3 0 00-2.4 1c-.6.7-.9 1.7-.9 3 0 1.4.3 2.5.9 3.2a3 3 0 002.4 1.1zm18.7-9.9V38h-1.6l-6.1-8v8h-2V26.6h1.6l6.1 8v-8zm11.5 8.7h-5.6l-1.1 2.7h-2l5-11.4h1.7l5.2 11.4h-2zm-.7-1.6l-2-4.9-2.2 5h4.2zm8-5.4h-3.8v-1.7h9.5v1.7h-3.7V38h-2zm8.6-1.7h7.7v1.6h-5.6v3.2h5.2V33H132v3.4h5.6V38h-7.7V26.6zm26.6 0V38h-1.6l-6.1-8v8h-2V26.6h1.6l6.1 8v-8zm4 0h2V38h-2zm17 0V38h-1.8v-7l-3 7h-1.3l-3-7v7h-1.9V26.6h1.6l4 9 3.8-9h1.7zM49.2 31l-4.6-8c-.4-.7-1-1-1.7-1h-9.3c-.7 0-1.3.3-1.7 1l-4.7 8c-.3.7-.3 1.3 0 2l4.7 8c.4.7 1 1 1.7 1h9.3c.7 0 1.3-.3 1.7-1l4.6-8c.4-.7.4-1.3 0-2z"/>
+  <defs id="defs74">
+    <radialGradient id="paint0_radial" cx="0" cy="0" r="1" gradientTransform="matrix(-212 0 0 -64 212 64)" gradientUnits="userSpaceOnUse">
+      <stop id="stop69" stop-color="#260133"/>
+      <stop id="stop71" offset="1" stop-color="#1F2348"/>
+    </radialGradient>
+  </defs>
+</svg>
```

Comment:
You're right 👍

Reply:
But now that I think about it, I'm not sure whether they actually work when used in an <img> tag. Likely not.

Reply:
No it will not work as an `img` tag. But maybe i can just import the SVGs as components like in `@nimiq/vue-components`. Then i'll be able to apply style to them since they'll be in the dom as actual SVGs.

Reply:
But if at the end the button doesn't have any hover effect then it doesn't make sense to add one on the donation app, in fact.

Reply:
Sure, having a hover effect in the donation app doesn't make sense if the actual button will not have  that.
I was thinking about using svg `<use>` tags. But apparently these do currently not support external svgs from other domains.
In the HTML snippet or maybe even markdown we could have a hover effect, but it's no necessity.
I think we're fine without it.